### ISSUE TITLE
Fails to detect some magic numbers

### ISSUE THREAD

**Issue by leonklingele** [tommy-muehle/go-mnd#9]
Here are some cases which `mnd` fails to detect:

```go
package main

import (
	"fmt"
)

func main() {
	ok()

	fail1()
	fail2()
}

func ok() {
	res := 42 * 2
	fmt.Println(res)
}

func fail1() {
	const c = 24
	res := c + (42 * 10)
	fmt.Println(res)
}

func fail2() {
	res := -12
	fmt.Println(res)
}
```

```sh
$ go vet -vettool $(which mnd) ./...
# mndtest
./main.go:14:9: Magic number: 42, in <operation> detected
./main.go:14:14: Magic number: 2, in <operation> detected
```

**Comment by tommy-muehle** [comment]
@leonklingele 
Thanks for the report! I'll fix it asap.

**Comment by tommy-muehle** [comment]
Fixed all missing cases in [v2.0.1](https://github.com/tommy-muehle/go-mnd/releases/tag/v2.0.1).
Thanks again for the report.
